app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    @reviews = @video.reviews.all
```

Comment:
For some reason, I think ``` .reload ``` didn't seem to work as expected when I first wrote this, which is why I changed to ``` .all ``` but I can't seem to recreate it at the moment, so will amend as you suggest!